site/components/footer/index.js

```diff
@@ -143,6 +143,8 @@ const Footer = () => (
 
           <a
             className={styles.mapLink}
+            target="_blank"
+            rel="noopener noreferrer"
```

Comment:
eslint complained without this, see blog post: https://mathiasbynens.github.io/rel-noopener/